source/extensions/transport_sockets/ssl/context_config_impl.cc

```diff
@@ -1,4 +1,4 @@
-#include "common/ssl/context_config_impl.h"
+#include "extensions/transport_sockets/ssl/context_config_impl.h"
```

Comment:
Ideally I think we want the namespaces to be consistent with the path, do you want to do that in this PR or in next PR? I'm fine either.